structuredheadings/plugin.js

```diff
@@ -130,6 +136,28 @@
 
       });
 
+      editor.addCommand("increaseHeadingLevel", {
+        startDisabled: true,
+        exec: function () {
+          var element = getCurrentBlock(editor.getSelection().getStartElement());
+          var nextElement = headerList[headerList.indexOf(element.getName()) + 1];
```

Comment:
Is order guaranteed in each browser?  MDN says Object.keys is in the same order as `for...in` which is supposed to be arbitrary.  Though I think in practice, it's not always.